daemon/graphdriver/aufs/aufs.go

```diff
@@ -573,7 +573,7 @@ func (a *Driver) aufsMount(ro []string, rw, target, mountLabel string) (err erro
 
 	offset := 54
 	if useDirperm() {
-		offset += len("dirperm1")
+		offset += len(",dirperm1")
```

Comment:
Ah yes, I found the 50 as well, but couldn't find a reference what it was based on